packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/SceneHierarchyTreeItem.tsx

```diff
@@ -49,8 +47,33 @@ const SceneHierarchyTreeItem: FC<SceneHierarchyTreeItemProps> = ({
   const { searchTerms } = useSceneHierarchyData();
   const isSearching = searchTerms !== '';
 
+  const isExpanded = () => {
+    return nodesToExpand.indexOf(key) > -1;
+  };
+
+  const removeNodeFromExpandList = (nodeRef: string) => {
+    if (nodesToExpand.indexOf(nodeRef) > -1) {
+      nodesToExpand.splice(nodesToExpand.indexOf(nodeRef), 1);
```

Comment:
It's not a good practice for the child to change the value of the prop passed from parent. the data flow should be top to down, not both.

Reply:
Code not required with the changed approach.